public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        @media (prefers-color-scheme: light) {
+            html {
+                background: rgb(244, 243, 248)
+            }
+        }
+        @media (prefers-color-scheme: dark) {
+            html {
+                background: rgb(9, 4, 26)
+            }
+        }
+      </style>
```

Comment:
Not seen this before ... I'm guessing it's for implementing mode toggle.  If so, looks good!